src/startupscriptgenerator/src/dotnetcore/scriptgenerator.go

```diff
@@ -69,6 +69,39 @@ func (gen *DotnetCoreStartupScriptGenerator) shouldApplicationInsightsBeConfigur
 	return false
 }
 
+// Checks if the .NET runtime version meets the version contraint
+func (gen *DotnetCoreStartupScriptGenerator) isDotnetRuntimeVersionMeetConstraint(versionConstraint string) bool {
+	dotNetRuntimeVersion := ""
+	
+	if gen.Manifest.DotNetCoreRuntimeVersion != "" {
+		dotNetRuntimeVersion = gen.Manifest.DotNetCoreRuntimeVersion
+	} else {
+		dotNetRuntimeVersion = os.Getenv("DOTNET_VERSION")
+	}
+	
+	fmt.Printf("\nDotNet Runtime %s", dotNetRuntimeVersion)
+	
+	if dotNetRuntimeVersion != "" {
+		dotNetRuntimeVersionConstraint, err := semver.NewConstraint(">= " + versionConstraint)
+		if err != nil {
+    		fmt.Printf("\nError in creating semver constraint %s", err)
+		}
+
+		dotNetCurrentVersion, err := semver.NewVersion(dotNetRuntimeVersion)
+		if err != nil {
+    		fmt.Printf("\nError in parsing current version to semver version %s", err)
+		}
+		// Check if the version meets the constraints. The a variable will be true.
+		constraintCheckResult := dotNetRuntimeVersionConstraint.Check(dotNetCurrentVersion)
+				
+	    if constraintCheckResult  {
+			   fmt.Printf("\nBefore returning true")
```

Comment:
nit: can we remove this message? In addition, if we don't need to log any additional information, then I'd replace this `if constraintCheckResult` block with the following:

```go
return constraintCheckResult
```

Reply:
Updated